06-building-a-program-start-to-finish/writing-tests/Output/inverted-index-spec.js

```diff
@@ -0,0 +1,180 @@
+/* global describe, it, expect */
```

Comment:
Please adhere to the preferred naming convention for this file. Consult with your peers